source/about/accessibility_terms.md

```diff
@@ -0,0 +1,44 @@
+# Accessibility Terms
+
+arXiv is committed to creating an accessible experience for all users. Below are useful descriptions and terms, our thinking behind this commitment, and our roadmap for reaching and sustainably maintaining an accessible platform for the entire arXiv community.
+{ .intro }
+
+## Important terms
+
+**Accessibility** = The practice of making a website functional for as many people as possible, including those using assistive technologies.
+
+**ADA** = The Americans with Disabilities Act of 1990 is established federal law. It was written before the first website was launched (arXiv is from 1991 and among the first) and lacks specific digital guidelines, but the Supreme Court and the Department of Justice have confirmed that the ADA applies to the websites of &quot;covered entities&quot; such as Cornell University and arXiv.
```

Comment:
I'm not sure this is so cut and dry, and I'd prefer to avoid making legal pronouncements publicly on our website.

Reply:
simplified.